models/DataObject/ClassDefinition/Data/ExternalImage.php

```diff
@@ -298,6 +298,20 @@ public function synchronizeWithMasterDefinition(Model\DataObject\ClassDefinition
         $this->inputWidth = $masterDefinition->inputWidth;
     }
 
+    /**
+     * @param DataObject\Data\ExternalImage|null $data
+     * @param bool $omitMandatoryCheck
+     * @param array $params
+     *
+     * @throws Model\Element\ValidationException
+     */
+    public function checkValidity($data, $omitMandatoryCheck = false, $params = [])
+    {
+        if ($this->getMandatory() && !$omitMandatoryCheck && !$this->isEmpty($data)) {
```

Comment:
throw exception when data is empty
```suggestion
        if ($this->getMandatory() && !$omitMandatoryCheck && $this->isEmpty($data)) {
```